spec/javascripts/modules/contact-embed-modal-spec.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env jasmine */
+/* global ContactEmbedModal, fetchMock, buildModalDialogue, removeModalDialogue */
+
+describe('ContactEmbedModal', function () {
+  'use strict'
+
+  var link, contactEmbedModal
+
+  beforeEach(function () {
+    buildModalDialogue()
```

Comment:
That seems a good shout. I avoided this as I thought it made the tests more realistic by using the instance variable but on review I don't think there are any where this would impact whether they pass or not.